src/components/ToggleSwitch/index.tsx

```diff
@@ -69,15 +78,15 @@ const styles = StyleSheet.create({
     })
   },
   labelLeft: {
-    marginRight: 5,
+    marginRight: dimen.normalizeDimen(5),
     color: '#FFFFFF',
-    fontSize: 12,
+    fontSize: normalizeFontSize(12),
     fontWeight: '400'
   },
   labelRight: {
-    marginLeft: 5,
+    marginLeft: dimen.normalizeDimen(5),
     color: '#FFFFFF',
-    fontSize: 12,
+    fontSize: normalizeFontSize(12),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `labelLeft` and `labelRight` styles are appropriate. Using the `dimen.normalizeDimen` function for the `marginRight` and `marginLeft` properties and the `normalizeFontSize` function for the `fontSize` property will ensure that the labels scale properly on different screen sizes.
```diff
+ marginRight: dimen.normalizeDimen(5),
+ fontSize: normalizeFontSize(12),
+ marginLeft: dimen.normalizeDimen(5),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->